packages/babel-plugin-transform-react-jsx-source/test/fixtures/react-source/basic-sample/expected.js

```diff
@@ -1,4 +1,5 @@
+var __jsxFileName = "/I/am/not/sure/how/to/get/path/to/test/fixtures/react-source/basic-sample/actual.js";
```

Comment:
I think we can do a test to run/evaluate code instead of expecting an output like mentioned in https://github.com/babel/babel/pull/3257#discussion_r50222242.

You can create an `exec.js` instead of `actual/expected` (search through the repo for some exec.js examples)